src/lwc/geFormRenderer/geFormRenderer.html

```diff
@@ -16,11 +15,18 @@
                         </ul>
                     </c-util-page-level-message>
                 </div>
-                <template for:each={sections} for:item='section'>
-                    <c-ge-form-section key={section.id} section={section}></c-ge-form-section>
-                </template>
-            </lightning-layout-item>
-        </lightning-layout>
+            <template if:true={isPermissionError}>
```

Comment:
I think we need to fix the indents from here through the rest of the template - this "template" block should be one additional level in, etc